examples/one-2-one/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@loopback/example-one-2-one",
+  "version": "1.3.1",
+  "description": "Continuation of the hasOne example using relations in LoopBack 4.",
```

Comment:
The description says "Continuation of the hasOne example" - which example do you mean by "hasOne example"?

Reply:
Hello @bajtos , thanks you for your comments. I will add shortly the test of the delete operation. Just for information I already completed the PATCH operation as well, and I will commit it on next week. About the one-2-one example: the example implementation is fitting the documentation in LB4 for hasOne (https://loopback.io/doc/en/lb4/hasOne-relation.html) that describes Supplier and Account (by the way same as in LB3). "Todo-List has one" could make confusion with the existing "Todo-List" that implement hasMany relation. In my opinion and from my own experience as loopback new learner, better to have each relation with its own example to ease the learning and fit the documentation.

Reply:
Makes sense 👍 

Let's find a new better example name please. How about `@loopback/example-hasOne-relation` or maybe just `@loopback/example-hasOne`?

Reply:
Or maybe `@loopback/example-supplier-account`?

Reply:
Let's use a clearer description. How about "An example showing hasOne relation in LoopBack 4"?